example/ios/Runner/Info.plist

```diff
@@ -42,11 +44,9 @@
 	<key>com.posthog.posthog.POSTHOG_HOST</key>
 	<string>https://app.posthog.com</string>
 	<key>com.posthog.posthog.API_KEY</key>
-	<string>YOUR_API_KEY_GOES_HERE</string>
+	<string>_6SG-F7I1vCuZ-HdJL3VZQqjBlaSb1_20hDPwqMNnGI</string>
```

Comment:
should this be hard-coded?

Reply:
It's just our example project within the repo, so you can run and test stuff right away, its one of our testing projects key.